### PR TITLE
Update Local deployment guide with connectivity topics

### PR DESCRIPTION
**Pull request by theburi**
## Description

My goal is to improve the local deployment guide to help users do E2E

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by christinaausley — CHANGES_REQUESTED** [review]
A few comments to clarify a few sections.

Additionally, I'm not sure we need to break this up into tabs. I think we can keep everything within a single file here, particularly given the trouble we've been having with deep-linking to tabbed content lately.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/local-kubernetes-cluster.md [hunk 1]
```diff
@@ -1,58 +1,202 @@
...
+[
+{label: 'Port-forwarding', value: 'port-forward' },
+{label: 'Ingress configuration', value: 'ingress' }
+]}>
+
+<TabItem value="port-forward">
+
+To interact with the Camunda services inside a Kubernetes cluster without Ingress setup, you can use the [kubectl port-forward](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_port-forward/) command to route traffic from your local machine to the services running in the kind cluster. This is useful for quick tests or for development purposes.
+
+First, port-forward each of the components. Use a separate terminal for each command. Then, each component can be accessed at `http://localhost:PORT` (for example, Optimize will have the address `http://localhost:8083`). A full list of port mapping contains in a Helm deployment status, for example, `helm status camunda-platform`.
```
[the remark this answers is not in the document]
Comment (theburi) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/local-kubernetes-cluster.md:90]: rephrased

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/local-kubernetes-cluster.md [hunk 2]
```diff
@@ -1,58 +1,202 @@
...
+    contextPath: "/tasklist"
+
+zeebe-gateway:
+ingress:
+    enabled: true
+    className: nginx
+    host: "zeebe.camunda.local"
+```
+
+Proceed to install the Camunda Helm chart [Deploy](#deploy)
```
[the remark this answers is not in the document]
Comment (theburi) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/local-kubernetes-cluster.md:196]: Yes, I slightly modified this sentence. Let me know if it works.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/local-kubernetes-cluster.md [hunk 3]
```diff
@@ -1,58 +1,202 @@
...
 
 - [kind](https://kind.sigs.k8s.io/docs/user/quick-start)
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
-- [helm](https://helm.sh/docs/intro/install/)
+- [Helm](https://helm.sh/docs/intro/install/)
 
 ## Create a local Kubernetes cluster
 
-If you have not already, create a local Kubernetes cluster with the following command:
+At this stage, you should consider how to connect to the future Camunda 8 cluster. If you are setting up Camunda 8 for the first time, we recommend using **port-forwarding**, in which case you may continue to create kind clusters.
```
[the remark this answers is not in the document]
Comment (theburi) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/local-kubernetes-cluster.md:24]: What do you mean?

**Comment by theburi** [comment]
> A few comments to clarify a few sections.
> 
> Additionally, I'm not sure we need to break this up into tabs. I think we can keep everything within a single file here, particularly given the trouble we've been having with deep-linking to tabbed content lately.

I find one large file gets newcomers confused as it's hard to follow, and ignore sections. Tabs create a single flow, hiding sections and avoiding confusion.
In this particular case, we would not link to these tabs as we have other articles explaining these topics in all detail.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/local-kubernetes-cluster.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/local-kubernetes-cluster.md:24]: Similar to linking to the other sections in the doc, I would link to "port-forwarding" and "ingress".

[the remark this answers is not in the document]
Comment (theburi) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/local-kubernetes-cluster.md]: I made a change and I added a link that seems to work on my local build.

**Comment by christinaausley** [comment]
Is this linked to a particular issue and is this for `next` or the `8.5` release only?

**Review by christinaausley — APPROVED** [review]

**Review by jessesimpson36 — APPROVED** [review]

**Review by aabouzaid — APPROVED** [review]
LGTM, thank you :+1:
